firebase-config.js

```diff
@@ -0,0 +1,11 @@
+require('dotenv').config();
+
+const firebaseConfig = {
+  scopes: ['https://www.googleapis.com/auth/calendar.readonly'],
+  email: process.env.GOOGLE_EMAIL,
+  channelCalendarId: process.env.CHANNEL_CALENDAR_ID
+};
```

Comment:
Tal vez falta actualizar el `.env.example`?

Reply:
Added this change